frontend/src/core/api/base.js

```diff
@@ -106,7 +106,7 @@ export default class APIBase {
         };
 
         if (isObject(results)) {
-          const newObj = {};
+          const newObj: Object = {};
```

Comment:
This should not be needed, Flow should be able to guess the data type from the assignment. Is it not the case?

Reply:
That's what I would think too, but I kept getting an error 🤷‍♀

Reply:
What's the error?

Reply:
``` 
Cannot call await with api.entity.getHistory(...) bound to p because:
 • Either cannot call receive with content bound to translations because
   object literal [1] is incompatible with array type [2].
 • Or cannot call receive with content bound to translations because
   Promise [3] is incompatible with array type [2].

     src/modules/history/actions.js
 [2]  28│     translations: Array<Object>,
        :
      70│         // Abort all previously running requests.
      71│         await api.entity.abort();
      72│
      73│         const content = await api.entity.getHistory(entity, locale, pluralForm);
      74│         dispatch(receive(content));
      75│     }
      76│ }

     /tmp/flow/flowlib_12cd5d1e/core.js
 [3] 623│ declare class Promise<+R> {

     src/core/api/base.js
 [1] 105│             const newObj = {};

Reply:
Eeeew. OK. Then let's do what you did but replace `Object` with `any`. 

:grimacing: Flow :grimacing: